pkg/logs/logs.go

```diff
@@ -61,23 +61,47 @@ func (writer GlogWriter) Write(data []byte) (n int, err error) {
 }
 
 // InitLogs initializes logs the way we want for kubernetes.
-func InitLogs(fs *flag.FlagSet) {
-	if fs == nil {
-		fs = flag.CommandLine
-	}
-	klog.InitFlags(fs)
-	_ = fs.Set("logtostderr", "true")
```

Comment:
Is there a reason we had that in the first place? I checked and `--logtostderr` already defaults to `true`, so I'd be OK with removing this line.

Reply:
I think this default value was changed upstream https://github.com/kubernetes/klog/issues/38